{{cookiecutter.project_shortname}}/invenio.cfg

```diff
@@ -34,6 +37,37 @@ SQLALCHEMY_DATABASE_URI="mysql+pymysql://{{cookiecutter.project_shortname}}:{{co
 {%- endif %}
 
 
+# Invenio-App
+# ===========
+# See https://invenio-app.readthedocs.io/en/latest/configuration.html
+
+APP_DEFAULT_SECURE_HEADERS = {
+    'content_security_policy': {
```

Comment:
I ran it with the current development workflow (but not with the react changes) and it works. I am fairly confident this will help with the react side of things since it passes `'unsafe-inline'` which allows code in `<script></script>` tags to work and inline styles. But try it out on your end for sure.